mne_bids/mne_bids.py

```diff
@@ -641,6 +641,19 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     kind = _handle_kind(raw)
 
     bids_fname = bids_basename + '_%s%s' % (kind, ext)
+
+    # check whether the info provided indicates that the data is emptyroom
+    # data
+    emptyroom = False
+    if subject_id == 'emptyroom' and task == 'noise':
+        session_id = datetime.fromtimestamp(
+            raw.info['meas_date'][0]).strftime('%Y%m%d')
```

Comment:
I would actually put this in the example instead of trying to bake it into `mne-bids`

Reply:
I thought we agreed to have it in `mne-bids`?
Although upon re-reading your comment I see that I misread it.
I still think it is nicer having it here as it makes it much easier for anyone using it, instead of having to have that code every time...

Reply:
I feel it would be too much magic to change the filename ...

Reply:
yes, I would do the 80-20 rule here. Thanks a lot!